### PR TITLE
Bug 929272

### PR DESCRIPTION
**Pull request by pemmasanikrishna**
Replies on reviews display, same label as regular reviews "edit review" and "delete review".
It should say "edit reply" and "delete reply",

### CODE REVIEW

File: apps/reviews/templates/reviews/review.html [hunk 1]
```diff
@@ -85,13 +85,13 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
     {% if review.user_id == request.user.id %}
       <li>
         <a class="review-edit" href="#">
-          {{ _('Edit review') }}</a>
+          {{ _('Edit reply') }}</a>
```
Comment (magopian) [inline apps/reviews/templates/reviews/review.html]: You need to make sure that it only says "reply" if it's a reply, not in all the cases. If you check a few lines above (on line 79), there's an example use of `is_reply`.

Using this, you should be able to display "reply" or "review" when it's relevant.

Let me know if you need more help!

**Comment by magopian** [comment]
Hello @pemmasanikrishna , are you going to work on this, or should I take it from here?

**Comment by pemmasanikrishna** [comment]
@magopian  is this ok ?

**Comment by pemmasanikrishna** [comment]
@magopian  should I use " {% if review.user_id == request.user.id and((is_reply or has_reply) and (perms.is_author or perms.is_admin))} "  condition in the line 85 ??

File: apps/addons/tests/test_views.py [hunk 2]
```diff
@@ -1335,7 +1335,7 @@ class TestAddonSharing(amo.tests.TestCase):
     def test_redirect_sharing(self):
         addon = Addon.objects.get(id=3615)
         r = self.client.get(reverse('addons.share', args=['a3615']),
-                            {'service': 'delicious'})
+                            {'service': 'facebook'})
```
Comment (magopian) [inline apps/addons/tests/test_views.py]: I don't think this change should be here, could you please revert it?

**Comment by magopian** [comment]
@pemmasanikrishna you shouldn't use this test on line 85. Instead, you should have a test that depending on `is_reply` will display either `{{ _('Edit review') }}` or `{{ _('Edit reply') }}`.

**Comment by kmaglione** [comment]
I don't want to pick at nits, but can you please put a description of the bug that you're fixing after the bug numbers in your commit summaries?

**Comment by kmaglione** [comment]
Also, can you rebase this so that it only includes commit 7da2280, and not the one for bug 780356, or the merge commit?

Thanks.

**Comment by pemmasanikrishna** [comment]
@magopian  Is this the way to do it ?

File: apps/reviews/templates/reviews/review.html [hunk 3]
```diff
@@ -83,16 +83,31 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
...
-      <li>
-        <a class="review-edit" href="#">
-          {{ _('Edit review') }}</a>
-      </li>
+      {% if is_reply %}
+        <li>
+          <a class="review-edit" href="#">
+            {{ _('Edit reply') }}</a>
+        </li>
+      {% elif not is_reply %}
```
Comment (magopian) [inline apps/reviews/templates/reviews/review.html]: You simply need to write `{% else %}` here, it's a boolean case, there's no other choice.

File: apps/reviews/templates/reviews/review.html [hunk 4]
```diff
@@ -83,16 +83,31 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
...
+        </li>
+      {% endif %}
     {% endif %}
     {% if check_review_delete(review) %}
-      <li>
-        <a class="delete-review" href="{{ url('addons.reviews.delete', addon.slug, review.id) }}">
-          {{ _('Delete review') }}</a>
-      </li>
+      {% if is_reply %}
+
```
Comment (magopian) [inline apps/reviews/templates/reviews/review.html]: remove this empty line

File: apps/reviews/templates/reviews/review.html [hunk 5]
```diff
@@ -83,16 +83,31 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
...
+        <li>
+          <a class="delete-review" href="{{ url('addons.reviews.delete', addon.slug, review.id) }}">
+            {{ _('Delete reply') }}</a>
+        </li>
+      {% elif not is_reply %}
+        <li>
+          <a class="delete-review" href="{{ url('addons.reviews.delete', addon.slug, review.id) }}">
+            {{ _('Delete review') }}</a>
+        </li>
+      {% endif %}
```
Comment (magopian) [inline apps/reviews/templates/reviews/review.html]: Remove the trailing spaces here at the end of the line

File: apps/reviews/templates/reviews/review.html [hunk 6]
```diff
@@ -83,16 +83,31 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
...
-        <a class="delete-review" href="{{ url('addons.reviews.delete', addon.slug, review.id) }}">
-          {{ _('Delete review') }}</a>
-      </li>
+      {% if is_reply %}
+    
+        <li>
+          <a class="delete-review" href="{{ url('addons.reviews.delete', addon.slug, review.id) }}">
+            {{ _('Delete reply') }}</a>
+        </li>
+      {% elif not is_reply %}
```
Comment (magopian) [inline apps/reviews/templates/reviews/review.html]: As above, you just need an `else` here.

**Comment by magopian** [comment]
This is looking good @pemmasanikrishna ! However, there's quite a few commits, and there's also code that isn't relevant to this bug (see my comment https://github.com/mozilla/olympia/pull/561#discussion_r34442637).

So, could you please create a new branch, with just the code changes to the template, with my latest comments, and reopen a new pull request for that? Thanks!

**Comment by kmaglione** [comment]
You've added even more merge commits and different versions of the same commit in this push. Do you need help rebasing/cherry-picking them away? Feel free to ping me on IRC (John-Galt in irc://irc.mozilla.org/amo).

Also, re my comment about your commit summary, I didn't realize that you had the description on separate lines. Can you at least add a short description to the first line, after the bug number, so it's visible in the changelog?

Oh, and thanks for doing this work :)

**Comment by magopian** [comment]
Closed in favor of #607

**Pull request closed without merging**